mne_sandbox/preprocessing/bads/bads.py

```diff
@@ -0,0 +1,207 @@
+# Authors: Denis Engemann <denis.engemann@gmail.com>
+#          Marijn van Vliet <w.m.vanvliet@gmail.com>
+# License: BSD (3-clause)
+
+import numpy as np
+from mne.utils import verbose
+from mne.io.pick import pick_info
+from mne.io.pick import pick_types
+from mne.defaults import _handle_default
+from . import faster_ as _faster
+
+
+@verbose
+def find_bad_channels(epochs, picks=None, method='faster', method_params=None,
+                      return_by_metric=False, verbose=None):
+    """Implements the first step of the FASTER algorithm.
```

Comment:
not very descriptive IMO - better to just write that it looks for bad channels according to (first step of) the FASTER algorithm